ResearchUXFactory/UIColor+Utilities.swift

```diff
@@ -56,22 +56,23 @@ open class SBAColorInfo : NSObject {
 extension UIColor {
     
     static public func color(for key: String) -> UIColor? {
-        return SBAColorInfo.defaultColorInfo.color(for: key)
+        return SBAColorInfo.shared.color(for: key)
     }
     
-    static public func primaryTintColor() -> UIColor? {
-        return SBAColorInfo.defaultColorInfo.color(for: "primaryTintColor")
-    }
+    /**
+     If the shared ColorInfo.plist includes a primary tint color then set the color
+     */
+    static public let primaryTintColor: UIColor? = SBAColorInfo.shared.color(for: "primaryTintColor")
     
-    static public func greenTintColor() -> UIColor? {
-        // For certain cases, we want to use a green checkmark and will override the default color
-        return SBAColorInfo.defaultColorInfo.color(for: "greenTintColor") ?? UIColor(hexString: "#44d24e")
-    }
+    /** 
+     For certain cases, we want to use a green checkmark and will override the default color
+    */
+    static public let greenTintColor: UIColor = SBAColorInfo.shared.color(for: "greenTintColor") ?? UIColor(hexString: "#44d24e")!
     
-    static public func blueTintColor() -> UIColor? {
-        // For certain cases, we want to use a blue tint and will override the default color
-        return SBAColorInfo.defaultColorInfo.color(for: "blueTintColor") ?? UIColor(red:0.132, green:0.684, blue:0.959, alpha:1.000)
-    }
+    /**
+     For certain cases, we want to use a blue tint and will override the default color
+     */
+    static public let blueTintColor: UIColor = SBAColorInfo.shared.color(for: "blueTintColor") ?? UIColor(red:0.132, green:0.684, blue:0.959, alpha:1.000)
```

Comment:
any particular reason for using a hex string for green and decimal rgba values for blue?